app/views/employments/AddEmploymentPayrollNumberFormView.scala.html

```diff
@@ -14,64 +14,87 @@
  * limitations under the License.
  *@
 
-@import includes.forms.{input, inputRadioGroupStyling, submit}
 @import includes.{cancelLink, ptaHeader}
 @import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
 @import uk.gov.hmrc.tai.forms.employments.AddEmploymentPayrollNumberForm
 @import uk.gov.hmrc.tai.viewModels.employments.PayrollNumberViewModel
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import scala.concurrent.ExecutionContext
 
-@this(mainTemplate: MainTemplate, form: FormWithCSRF)
+@this(main: MainTempleteNew, form: FormWithCSRF, govukRadios: GovukRadios, govukButton: GovukButton, govukInput : GovukInput, govukErrorSummary: GovukErrorSummary)
 
 @(payrollNumberForm: Form[AddEmploymentPayrollNumberForm], payrollNumberViewModel: PayrollNumberViewModel)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
 
 @header = @{
     ptaHeader(
-        formForErrorSummary = Some(payrollNumberForm),
-        displayBackLink = true,
+        displayBackLink = false,
         mainHeadingText = Messages("tai.addEmployment.employmentPayrollNumber.title", payrollNumberViewModel.employmentName),
-        preHeadingText = Messages("add.missing.employment"))}
+        preHeadingText = Messages("add.missing.employment"),
+        showGdsVersion = true
+    )}
 
-@mainTemplate(
+@main(
+    pageTitle = Some(Messages("tai.currentYearSummary.heading")),
     title = Messages("tai.addEmployment.employmentPayrollNumber.pagetitle"),
     authedUser = Some(user),
-    pageHeader = Some(header),
-    articleClasses = None
+    backLinkContent = Messages("tai.back-link.upper")
 ) {
+    <div class = "govuk-grid-row">
+        <div class = "govuk-grid-column-two-thirds">
+            @if(payrollNumberForm.errors.nonEmpty) {
+                @govukErrorSummary(ErrorSummary(errorList = payrollNumberForm.errors.asTextErrorLinks, title = Text(messages("tai.income.error.form.summary"))))
+            }
+            @header
```

Comment:
do we need header ? - As the H1 heading is displayed in the legend on these journeys with forms ?

Reply:
the only problem I see is that this view has the header on top of the list, should the list go as part of the Fieldset?